spotirec.py

```diff
@@ -307,24 +310,24 @@ def remove_from_blacklist(entries: list):
         with open(blacklist_path, 'r') as file:
             blacklist = json.loads(file.read())
     except json.decoder.JSONDecodeError:
-        print('Error: blacklist is empty')
+        logging.error('Error: blacklist is empty')
```

Comment:
'Error:' should probably be removed, if you manage to suppress the `LEVEL:root:` part of the log messages.

Reply:
Agreed. I think a good form could be to have INFO and above have no additional formatting and any _worse_ be prefixed by log-level.

Reply:
Sounds good to me.